src/Our.Umbraco.StackedContent/Web/Helpers/ViewHelper.cs

```diff
@@ -1,15 +1,36 @@
 ﻿using System.IO;
+using System.Linq;
 using System.Web;
 using System.Web.Mvc;
 using System.Web.Routing;
+using Umbraco.Core.Logging;
 
 namespace Our.Umbraco.StackedContent.Web.Helpers
 {
-    internal static class ViewHelper
+    public static class ViewHelper
```

Comment:
If we are making this public, I wonder if we should name it something more stacked content related? `StackedContentViewHelper`? Or is the namespace enough? @leekelleher thoughts?

Reply:
That was my thought at least, that the namespace would be enough.

Reply:
If @leekelleher is happy as well, then I'm happy :+1:

Reply:
Yeah, I had the same thought - the dreaded "are we committing to this API?" Switching from `internal` to `public`.
I agree, the namespace is covers this 👍

Reply:
I made sure that the AddViewLocationFormats method is the only thing not still private or internal, so we continue to only expose what is needed.